Storage/src/Lifecycle.php

```diff
@@ -90,7 +90,8 @@ public function __construct(array $lifecycle = [])
      *     @type string[] $matchesStorageClass Objects having any of the storage
      *           classes specified by this condition will be matched. Values
      *           include `"MULTI_REGIONAL"`, `"REGIONAL"`, `"NEARLINE"`,
```

Comment:
It would be good to re-sort this list as follows: STANDARD, NEARLINE, COLDLINE, ARCHIVE. Then note that MULTI_REGIONAL, REGIONAL, DURABLE_REDUCED_AVAILABILITY are legacy (ie not recommended for new implementations). Should look similar to the comment here in the ruby client library, plus the addition of ARCHIVE: https://github.com/googleapis/google-cloud-ruby/blob/master/google-cloud-storage/lib/google/cloud/storage/bucket.rb#L358 . You can also add a link to the docs about storage classes here (if you don't think it's too much info for this context): https://cloud.google.com/storage/docs/storage-classes

Same thing for StorageClient.php

Reply:
Done!